test/framework/src/main/java/org/elasticsearch/test/rest/yaml/Features.java

```diff
@@ -39,6 +39,7 @@
             "catch_unauthorized",
             "embedded_stash_key",
             "headers",
+            "stash_in_key",
             "stash_in_path",
```

Comment:
I think only `stash_in_key` is limited to doc tests. The others are in use in other spots.